render.yaml

```diff
@@ -0,0 +1,18 @@
+services:
+- type: pserv
+  name: pgbouncer
+  env: docker
+  plan: starter plus
+  repo: https://github.com/renderinc/docker-pgbouncer
```

Comment:
This redirects to https://github.com/render-oss/docker-pgbouncer, maybe use the new URL directly?